### PR TITLE
Arm backend: Add support for non-folded quantized BatchNorm

### PR DESCRIPTION
**Pull request by tom-arm**
* Support quantized batch norm when it follows an operator that it cannot be folded into.

Change-Id: I55ef21492c521cddbd8282ad9bfe17f0e2c573d8

cc @digantdesai @freddan80 @per @zingo @oscarandersson8218 @mansnils @Sebastian-Larsson @robell @rascani

### CODE REVIEW

**Comment by tom-arm** [comment]
Failures unrelated

**Comment by tom-arm** [comment]
Fix for this issue: https://github.com/pytorch/executorch/issues/17241

**Review by zingo — APPROVED** [review]
